staging/src/k8s.io/kube-aggregator/pkg/client/informers/externalversions/apiregistration/interface.go

```diff
@@ -26,10 +26,10 @@ import (
 
 // Interface provides access to each of this group's versions.
 type Interface interface {
-	// V1beta1 provides access to shared informers for resources in V1beta1.
-	V1beta1() v1beta1.Interface
 	// V1 provides access to shared informers for resources in V1.
 	V1() v1.Interface
+	// V1beta1 provides access to shared informers for resources in V1beta1.
+	V1beta1() v1beta1.Interface
```

Comment:
Are the ordering changes still deterministic and changed just because we're sorting/scanning differently now? Wanted to make sure this order won't flutter